extensions/amp-story/1.0/animation-presets.js

```diff
@@ -35,6 +35,23 @@ const SCALE_HIGH_DEFAULT = 3;
 /** @const {number} */
 const SCALE_LOW_DEFAULT = 1;
 
+/** @const {string} */
+const SCALE_START_ATTRIBUTE_NAME = 'scale-start';
+/** @const {string} */
+const SCALE_END_ATTRIBUTE_NAME = 'scale-end';
+/** @const {string} */
+const TRANSLATE_X_ATTRIBUTE_NAME = 'translate-x';
+/** @const {string} */
+const TRANSLATE_Y_ATTRIBUTE_NAME = 'translate-y';
+
+/** @const {!Array<string>} */
+export const PRESET_OPTION_ATTRIBUTES = [
```

Comment:
If I understand correctly you're suggesting doing:

```js
/** @enum {string} */
export PresetOptionAttributes = {
  translateY: 'translate-y',
};

const translateY = options[PresetOptionAttributes.translateY];

// Object.values is not allowed due to browser support
objectValuesEquivalent(PresetOptionAttributes).forEach(attribute => {})
```

rather than:

```js
const TRANSLATE_Y_ATTRIBUTE_NAME = 'translate-y';

/** @enum {string} */
export PRESET_OPTION_ATTRIBUTES = [
  TRANSLATE_Y_ATTRIBUTE_NAME,
];

const translateY = options[TRANSLATE_Y_ATTRIBUTE_NAME];

PRESET_OPTION_ATTRIBUTES.forEach(attribute => {})
```

So I can see how the first option contains the definition better, but then you require that `Object.values` call. I'd prefer to keep the definition static instead.